chrome_extension/background.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+// TODO: Generate an ECDSA key pair, and use the private key for authentication and the public key
+// as Oak label.
+//
+// See https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/sign
+
+// Maps a tab id to a boolean signalling whether that tab has entered Oak mode.
+//
+// Only contains entries for which the value is `true`.
+const tab_status = {};
```

Comment:
Personally I'd make this a map. :) 

The reason being that when looking up properties on an object, JavaScript goes up all the way up the prototype chain. This is less performant, but more importantly can lead to unexpected results. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

Map also exposes a `has` [method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/has), which is more predictable than reading an object property and evaluating it as boolean.

Reply:
Great, thanks!